src/component/ColorCard/index.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+const CardWrapper = styled.div`
+  width: 300px;
+  height: 430px;
+  border: 1px black solid;
+  border-radius: 10px;
+  margin: 10px;
+  margin-left: 10px;
+  margin-right: 0px;
+  display: inline-block;
+  //
+`;
+
+const ColorWrapper = styled.div`
+  height: 350px;
+  border: 1px solid black;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  background-color: ${(props) => (props.color ? props.color : "red")};
+`;
+
+const Textsection = styled.div`
+  height: 80px;
+  width: 300px;
```

Comment:
```suggestion

const TextSection = styled.div`
  height: 80px;
  width: 300px;
```

컴포넌트 네이밍에 PascalCase 를 적용해볼까요?